src/core/grabber/include/manipulator/modifier_flag_manager.hpp

```diff
@@ -21,6 +21,7 @@ class modifier_flag_manager final {
     states_[static_cast<size_t>(krbn::modifier_flag::right_option)] = std::make_unique<state>("option", "⌥");
     states_[static_cast<size_t>(krbn::modifier_flag::right_command)] = std::make_unique<state>("command", "⌘");
     states_[static_cast<size_t>(krbn::modifier_flag::fn)] = std::make_unique<state>("fn", "fn");
+    states_[static_cast<size_t>(krbn::modifier_flag::hyper)] = std::make_unique<state>("hyper", "hyper");
```

Comment:
No, I meant an option that says: "When I press `control_l` send `control_l + command_l + shift_l + option_l` to the system".